Inserted the full path to jcathy.jar file in Debian systems
--- a/jcathy
+++ b/jcathy
@@ -41,12 +41,7 @@
 
 if [[ $VERSION > 1.6 || $VERSION = 1.6 ]]; then
 	echo "jCathy is loading..."
-
-	# Edit to following line prepending to jcathy.jar the full path
-	# where you have installed that file. A common path in Debian
-	# system can be /usr/share/java/ and the whole command will become
-	# $JAVA -jar /usr/share/java/jcathy.jar
-	$JAVA -jar jcathy.jar
+	$JAVA -jar /usr/share/java/jcathy.jar
 else
 	echo -e "jCathy needs java version >= 1.6 (your version is $FULL_VERSION)"
 	exit 1
